android/src/main/java/com/reactcommunity/rndatetimepicker/RNDatePickerDialogFragment.java

```diff
@@ -134,7 +143,13 @@ static DatePickerDialog createDialog(
       c.set(Calendar.MINUTE, 0);
       c.set(Calendar.SECOND, 0);
       c.set(Calendar.MILLISECOND, 0);
-      datePicker.setMinDate(c.getTimeInMillis());
+
+      if (timeZoneOffsetInMilliseconds != null) {
+        int offset = TimeZone.getDefault().getOffset(c.getTimeInMillis()) - timeZoneOffsetInMilliseconds;
```

Comment:
can you please extract this into a function `int getOffset()` that returns 0 or `TimeZone.getDefault().getOffset(c.getTimeInMillis()) - timeZoneOffsetInMilliseconds`?

then reuse the fn here and for datePicker.setMaxDate()

thank you :)